app/assets/javascripts/app/project/project_sidebar.js

```diff
@@ -72,3 +74,79 @@ App.views.Project.addChild('ProjectSidebar', {
   }
 });
 
+
+App.views.Project.addChild('ProjectSidebarMobile', {
+  el: '#project-sidebar-mobile',
+
+  events:{
+    "click .show_reward_form": "showRewardForm",
+    "click .card-reward" : "selectReward"
+  },
+
+  selectReward: function(event){
+    var url = this.$(event.currentTarget).data('new_contribution_url');
+    this.navigate(url);
+    return false;
+  },
+
+  activate: function() {
+    this.$rewards = this.$('#rewards');
+    console.log(this.$rewards);
+    this.sortableRewards();
+    this.reloadRewards();
+
+  },
+
+  reloadRewards: function() {
+    var that = this;
+    $.get(that.$rewards.data('index_path')).success(function(data){
+      that.$rewards.html(data);
+    });
+  },
+
+  sortableRewards: function() {
+    if(this.$rewards.data("can_update") == true){
```

Comment:
`if(this.$rewards.data("can_update")) ` ou `if(this.$rewards.data("can_update") === true)`